docs/src/developer/overview.rst

```diff
@@ -0,0 +1,14 @@
+********
+Overview
+********
+
+Welcome to the project. We're excited you decided to improve Oríon!
+
+Hopefully, by then you should be familiar with our `contribution guide <https://github.com/Epistimio/orion/blob/master/CONTRIBUTING.md>`_ and `code of conduct <https://github.com/Epistimio/orion/blob/master/CODE_OF_CONDUCT.md>`_. We can jump straight into getting you started for your first contribution.
+
+- Dev installation instructions
+- Check everything is ready for some dev (Run the test suite!)
+- Coding Conventions
+- Testing
```

Comment:
These pages are to come in another PR right?

Reply:
So I was thinking of using the same PR, that's why it's a draft for now. I really don't want to have incomplete documentation pushed.

So there is two solutions:
1) Write a minimalist guide in overview.rst with link to the already existing doc
2) On the same pull request, finish the overhaul of the developer documentation

What do you prefer?

Reply:
Unless we have something different in mind, the documentation for the contributions will take much more time and effort than the CONTRIBUTING.md document. As long as the documentation being merged is in develop it is fine if parts are missing. The tradeoff is between too large PR to review and risk of missing parts when breaking down a PR in smaller ones that are merged independently. The way I see it, CONTRIBUTING.md referring to an incomplete overview.rst is fine since we know this is the next part you will be working on. From my experience the risk of low quality review on large PR is more serious than the risk of missing details when breaking down a project in small PRs.

Reply:
I completely agree with this point of view. I'll then tidy up the `overview.rst` and submit the PR for review :+1:.